core/src/test/java/org/apache/accumulo/core/tabletserver/log/LogEntryTest.java

```diff
@@ -64,7 +63,7 @@ public void testFromMetadata() {
   private void verifyLogEntry(LogEntry logEntry, Text expectedColumnQualifier) {
     assertEquals(validPath, logEntry.toString());
     assertEquals(validPath, logEntry.getPath());
-    assertEquals(HostAndPort.fromString(validHost.replace('+', ':')), logEntry.getTServer());
+    assertEquals(AddressUtil.parseAddress(validHost), logEntry.getTServer());
```

Comment:
I think this one might be okay to leave alone... that way the test validates the conversion is being done. With this change, it merely validates that it's consistent with AddressUtil... but AddressUtil could be wrongly used, and this would go undetected. It's probably best to keep the lower-level conversion in the test, so it's not dependent on the AddressUtil behavior.

Reply:
Reverted in 952b4f4b4e